lib/help.js

```diff
@@ -0,0 +1,141 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: generator-loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+var g = require('strong-globalize')();
+var _ = require('lodash');
+var table = require('text-table');
+var helpers = require('./helpers.js');
+
+/**
+ * Customize the help message based on how the generator is invoked (slc|yo)
+ * @param {Object} generator The generator instance
+ * @param {String} cmd The cmd name
+ * @returns {String} The customized help string
+ */
+exports.customHelp = function(generator, cmd, usageFilePath) {
```

Comment:
Can we change the signature to `function(generator, usageFilePath, cmd)`, so that callers can omit `cmd` instead of passing `null`?

```js
// ugly
return helpText.customHelp(this, null, 'loopback_datasource_usage.txt');

// better
return helpText.customHelp(this, 'loopback_datasource_usage.txt');
```